eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -210,6 +210,10 @@ jobs:
   - checkout: self
     submodules: recursive
 
+  - template: /eng/common/pipelines/templates/steps/verify-links.yml
```

Comment:
This step should run once in the [`GenerateReleaseArtifacts` job](https://github.com/Azure/azure-sdk-for-c/blob/97f06822468446fb3313558fc2166e10923f84a5/eng/pipelines/templates/jobs/archetype-sdk-client.yml#L284), not the `Validate` job.